server.go

```diff
@@ -813,15 +813,6 @@ func (l *listenSocket) Close() error {
 // Serve returns when lis.Accept fails with fatal errors.  lis will be closed when
 // this method returns.
 // Serve will return a non-nil error unless Stop or GracefulStop is called.
-//
-// Note: As of Go 1.21, the standard library overrides the OS defaults for
```

Comment:
Why is this deleted?

Reply:
Because I will be sending a follow-up PR to fix things on the server-side.

Reply:
Updated this with the latest commit, based on our discussion on the issue.